src/main/resources/application.yaml

```diff
@@ -7,7 +7,7 @@ management:
 
 redis:
   enabled: ${REDIS_ENABLED:true}
-  host: ${REDIS_HOST:redis://localhost}
+  host: ${REDIS_HOST:localhost}
```

Comment:
if i understood correctly, i think we can provide `redis://send-letter-redis` here as a default, similarly as in postgres configuration

Reply:
note that this property is called `host` not `url` or `address`

postgres config has this:
`url: jdbc:postgresql://${DRAFT_STORE_DB_HOST:localhost}:${DRAFT_STORE_DB_PORT:5432}/draftstore`

Reply:
ah. so `send-letter-redis` just?

Reply:
yup, full url with port is being built in [`RedisConfiguration.java`](https://github.com/hmcts/send-letter-service/pull/14/files#diff-d53b6cbae124f862a72d9f70a7c6a6e7)